content/en/methods/apps.md

```diff
@@ -83,29 +141,37 @@ GET /api/v1/apps/verify_credentials HTTP/1.1
 
 Confirm that the app's OAuth2 credentials work.
 
-**Returns:** [Application]({{< relref "entities/application" >}}), but without `client_id` or `client_secret`\
-**OAuth level:** App token + `read`\
+**Returns:** [Application]({{< relref "entities/application" >}})\
+**OAuth:** App token\
 **Version history:**\
 2.0.0 - added\
-2.7.2 - now returns `vapid_key`
+2.7.2 - now returns `vapid_key`\
+4.3.0 - deprecated `vapid_key`, please see [api/v2/instance]({{< relref "methods/Instance#v2">}})\
+4.3.0 - removed needing `read` scope to access this API, now any valid App token can be used\
+4.3.0 - added `scopes` and `redirect_uris` properties
```

Comment:
I think the 4.3.0 points should be combined into a single point

```
4.3.0 - `read` scope is no longer needed. Added `scopes` and `redirect_uris` properties; removed `vapid_key` (see [api/v2/instance]({{< relref "methods/Instance#v2">}}) instead)
```

also was vapid_key deprecated or removed? i don't see it in the example response below

Reply:
`vapid_key` is deprecated, pending removal in 5.0, I think I prefer keeping these as separate points, since these are all significant changes. Normally we don't have so many changes in a single endpoint for a single release.